### PR TITLE
Doc day runbook

### PR DESCRIPTION
**Pull request by melanierogan**
## What?

Updated the `RUNBOOK.md` with more relevant and up-to-date information.

## Why?

As a part of our team docs day to improve our SOS score.

## Scope and particulars of this PR (Please tick all that apply)
- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [x] Documentation / Runbook update
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)
- [ ] Review not required

## Checklist for **reviewers** (Please tick all that apply)
- [ ] Would a face-to-face review work better for this PR (Check if you feel you don't fully understand the PR)?
- [ ] Has the documentation been updated?

### CODE REVIEW

File: RUNBOOK.md [hunk 1]
```diff
@@ -28,3 +28,75 @@ reliability-engineering
 ## Supported By
 
 reliability-engineering
+
+## Known About By
+
+-   rhys.evans
+-   dora.militaru
+-   sarah.wells
```
Comment (wheresrhys) [inline RUNBOOK.md]: does sarah really know about it? (Not to do Sarah down, but this field is supposed to be for people who have some knowledge of the detail... and yes I know it's not a very good field name... we should look into ways to evolve it)

File: RUNBOOK.md [hunk 2]
```diff
@@ -28,3 +28,75 @@ reliability-engineering
...
+## First Line Troubleshooting
+
+Check runbook.md logs on [Splunk](https://financialtimes.splunkcloud.com/en-US/app/search/runbooksmd?form.field1.earliest=-60m%40m&form.field1.latest=now). This highlights all of the recent errors in production. Check if there is a recent build that failed using [CircleCI](https://circleci.com/gh/Financial-Times/runbook.md).
+
+## More Information
+
+Runbook.md is a toolchain used for improving the quality of runbooks across the business, and ensuring consistency of the information contained within runbooks with biz-ops – the knowledge base which models our business operations.
+Contact [#runbooks-in-repos](https://financialtimes.slack.com/messages/CFR0GPCAH) on Slack for more information.
+
+## Bespoke Monitoring
```
Comment (wheresrhys) [inline RUNBOOK.md]:
```suggestion
## Monitoring
```

Curious where you got the Bespoke Monitoring heading from?

File: RUNBOOK.md (hunk 1 above)
Comment (melanierogan) [reply RUNBOOK.md]: Fair point! I've removed her :)

File: RUNBOOK.md (hunk 2 above)
Comment (melanierogan) [reply RUNBOOK.md]: I was on a copy-paste spree, I _think_ I took it from another repo as their runbook was quite good, can't remember which one. I've updated the label now

**Pull request merged**